tools/test.py

```diff
@@ -1385,6 +1396,9 @@ def BuildOptions():
   result.add_option('--repeat',
       help='Number of times to repeat given tests',
       default=1, type="int")
+  result.add_option('--abort-on-timeout',
+    help='Send SIGABRT instead of SIGTERM to kill processes that time out',
+    default=False, dest="abort_on_timeout")
```

Comment:
We don't.  We don't maintain much python code, though.  Pointing out the occasional style issue isn't much of a burden.